dvc/remote/ssh/__init__.py

```diff
@@ -47,9 +50,21 @@ class RemoteSSH(RemoteBASE):
     # We use conservative setting of 4 instead to not exhaust max sessions.
     CHECKSUM_JOBS = 4
 
+    # XXX: Friendly reminder to add "reflink" to the default types
+    # after is implemented
+    DEFAULT_CACHE_TYPES = ["symlink", "copy"]
+
     def __init__(self, repo, config):
         super(RemoteSSH, self).__init__(repo, config)
 
+        types = config.get(Config.SECTION_CACHE_TYPE, None)
```

Comment:
Let's define DEFAULT_CACHE_TYPES = ["copy"] in RemoteBASE, since it is indeed what we use by default. And then move this `types` parsing to RemoteBASE as well, so we can avoid duplication in RemoteLOCAL and RemoteSSH.